seqr/views/apis/anvil_workspace_api.py

```diff
@@ -176,7 +180,14 @@ def create_project_from_workspace(request, namespace, name):
         logger.error('Uploading sample IDs to Google Storage failed. Errors: {}'.format(str(ee)), request.user,
                      detail=sample_ids)
 
-    # Send a slack message to the slack channel
+    # use airflow api to trigger AnVIL dags
+    try:
```

Comment:
it would be cleaner if you moved all the error handling code, including the slack message, into the `_trigger_data_loading` function so the control flow here is cleaner